spec/domain/mailing_lists/bulk_mail/imap_mail_validator_spec.rb

```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+
+# Copyright (c) 2012-2021, Hitobito AG. This file is part of
+# hitobito and licensed under the Affero General Public License version 3
+# or later. See the COPYING file at the top-level directory or at
+# https://github.com/hitobito/hitobito.
+
+require 'spec_helper'
+
+describe MailingLists::BulkMail::ImapMailValidator do
+  include MailingLists::ImapMailsHelper
+
+  let(:mailing_list) { mailing_lists(:leaders) }
+  let(:plain_body_mail) { imap_mail(plain_body: false) }
```

Comment:
maybe you could always use imap_mail to work with in the cases.
so add a let(:imap_mail) ... here (rename plain body mail)
if in one case it's not a plain body mail ... then override it. 

btw ... you call this variable plain_body_mai ... but plain_body is false ... ?